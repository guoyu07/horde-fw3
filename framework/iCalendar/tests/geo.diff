001- array (
001+ Fatal error: Cannot redeclare class Horde_iCalendar in /usr/share/php/Horde/iCalendar.php on line 26
002-   'latitude' => -17.87,
003-   'longitude' => 37.24,
004- )
005- array (
006-   'latitude' => 37.386013,
007-   'longitude' => -122.082932,
008- )