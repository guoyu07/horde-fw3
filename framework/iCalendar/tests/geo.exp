array (
  'latitude' => -17.87,
  'longitude' => 37.24,
)
array (
  'latitude' => 37.386013,
  'longitude' => -122.082932,
)