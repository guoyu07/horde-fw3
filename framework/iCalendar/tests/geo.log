
---- EXPECTED OUTPUT
array (
  'latitude' => -17.87,
  'longitude' => 37.24,
)
array (
  'latitude' => 37.386013,
  'longitude' => -122.082932,
)
---- ACTUAL OUTPUT
Fatal error: Cannot redeclare class Horde_iCalendar in /usr/share/php/Horde/iCalendar.php on line 26
---- FAILED
