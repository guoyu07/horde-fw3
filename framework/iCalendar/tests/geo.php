<?php

require_once dirname(__FILE__) . '/../iCalendar.php';
$ical = new Horde_iCalendar();

$data = 'BEGIN:VCARD
VERSION:2.1
GEO:37.24,-17.87
END:VCARD';

$ical->parseVCalendar($data);
$vcard = $ical->getComponent(0);
var_export($vcard->getAttribute('GEO'));
echo "\n";

$data = 'BEGIN:VCARD
VERSION:3.0
GEO:37.386013;-122.082932
END:VCARD';

$ical->parseVCalendar($data);
$vcard = $ical->getComponent(0);
var_export($vcard->getAttribute('GEO'));

?>
