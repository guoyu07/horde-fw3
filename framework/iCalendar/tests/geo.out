Fatal error: Cannot redeclare class Horde_iCalendar in /usr/share/php/Horde/iCalendar.php on line 26